### PR TITLE
chore: Update dropdown z-index to be on top of contact details pane

### PR DESCRIPTION
**Pull request by fayazara**
Fixes z index pf dropdown pane on top of contact details panel

Screenshot 18-11 at 14

### CODE REVIEW

**Review by iamsivin — DISMISSED** [review]

File: app/javascript/dashboard/assets/scss/plugins/_dropdown.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
   @include border-light;
   box-sizing: content-box;
   width: fit-content;
-  z-index: 999;
+  z-index: 9999;
```
Comment (nithindavid) [inline app/javascript/dashboard/assets/scss/plugins/_dropdown.scss:6]: 'app/javascript/shared/assets/stylesheets/z-index.scss' this has some defined variables. Can you try if it's possible with this ?

**Review by nithindavid — DISMISSED** [review]
Please see the minor comment.

File: app/javascript/dashboard/assets/scss/plugins/_dropdown.scss (hunk 1 above)
Comment (fayazara) [reply app/javascript/dashboard/assets/scss/plugins/_dropdown.scss]: Replaced the hardcoded value with a css var

**Review by muhsin-k — APPROVED** [review]

**Review by pranavrajs — APPROVED** [review]

**Review by nithindavid — APPROVED** [review]

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please check this with commandbar open.

**Comment by fayazara** [comment]
@pranavrajs command bar had the same z-index, updated now.

File: app/javascript/shared/assets/stylesheets/z-index.scss [hunk 2]
```diff
@@ -7,5 +7,6 @@
   --z-index-high: 1000;
   --z-index-higher: 2000;
   --z-index-much-higher: 5000;
+  --z-index-very-high: 9999;
```
Comment (pranavrajs) [inline app/javascript/shared/assets/stylesheets/z-index.scss:11]: 😆 We are running out of variable names.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**